cdap-ui/app/cdap/components/Reports/Reports.scss

```diff
@@ -14,11 +14,19 @@
  * the License.
  */
 
+@import '../../styles/variables.scss';
+
 $header-height: 70px;
+$header-bg-color: white;
+$bg-color: $grey-08;
 
 .reports-container {
+  height: calc(100vh - 50px - 54px);
```

Comment:
This is already defined in the `#app-container`(from common.scss) rule right? do we need this here as well?